### PR TITLE
Testing update eoan

### LINKED ISSUE #343: Update TESTING to reflect updated gtk-theme and gnome-shell testing
**Distribution (run `cat /etc/os-release`):**
Pop!_OS 19.10

**Actual issue**
TODO: Let us update the TESTING doc to reference new changes in the gtk-theme as well as things that should be looked for in the shell. This doc should outline the testing that, once completed/passed we should feel confident enough to release the theme for 19.10.

### PR DESCRIPTION
**Pull request by isantop**
Updates the TESTING.md to reflect changes in the design of the theme.

I'm only requiring one approval on this PR (pending successful CI) due to only affecting internal Github-related files.

Fixes #343

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -24,58 +24,67 @@ Things to look at:
...
+  - Pop-Dark
+  - Pop-Light
  
  **Other things to note:**
- 
- - Everything has a shadow
  - Everything needs to be legible/high contrast
- - Bright colors and title bars all have an inactive shade (when window is not selected)
- - Compact variants need to shrink most padding and sizes.
+  - Shooting for 7.5:1 contrast ratios
```
Comment (brs17) [inline TESTING.md]: What is 7.5:1 contrast ratio? How would one confirm such a ratio?

File: TESTING.md [hunk 2]
```diff
@@ -24,58 +24,67 @@ Things to look at:
...
  - Save buttons once they can be clicked
  - Affirmative/progressive action buttons/controls
  
  *Header and title bars:* 
- - brown in normal theme
- - dark theme - darker brown
- - light theme - grey
+ - brown in normal theme (#574F4A)
+ - dark theme - dark grey (#363636)
+ - light theme - light grey (Light theme is currently missing)
```
Comment (brs17) [inline TESTING.md]: If it is currently missing, why don't we leave it out until such a theme is ready and then make a pr with those additions?

File: TESTING.md (hunk 1 above)
Comment (isantop) [reply TESTING.md]: Addressed on line 104

File: TESTING.md (hunk 2 above)
Comment (isantop) [reply TESTING.md]: Updated

**Pull request merged**